pages/api/v1/fantasy/index.ts

```diff
@@ -0,0 +1,87 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import SQL from 'sql-template-strings';
+import Cors from 'cors';
+import { query } from '../../../../lib/db';
+import use from '../../../../lib/middleware';
+
+const cors = Cors({
+  methods: ['GET', 'HEAD'],
+});
+
+export interface FantasyData {
+  seasonID: number;
+  position: string;
+  name: string;
+  fantasyPoints: number;
+}
+
+export default async (
+  req: NextApiRequest,
+  res: NextApiResponse<Array<FantasyData> | string>
+): Promise<void> => {
+  await use(req, res, cors);
+
+  const { season = null, position = null, name = null} = req.query;
+
+  const search = SQL`SELECT * FROM fantasypoints`
+    .append(
+      season != null || position != null || name != null
+        ? SQL`
+        WHERE  
+        `
+        : ''
+    ).append(
+      season != null
+        ? SQL`
+        seasonID=${season}
+        `
+        : ''
+    ).append(
+      season != null && position != null
+        ? SQL`
+         AND 
+        `
+        : ''
+    ).append(
+      position != null
+        ? SQL`
+        position=${position}
+        `
+        : ''
+    ).append(
```

Comment:
If season is predefined as in my last review, we should be able to have it so we can just add `AND position=${position}` or whatever. That way we can get rid of some of these appends which make our code slower.

Reply:
Same thing with the `name` conditions